pkgs/development/python-modules/cartopy/default.nix

```
@@ -0,0 +1,38 @@
+{ buildPythonPackage, lib, fetchPypi,
+  pytest, filelock, mock, pep8,
+  cython,
+  six, pyshp, shapely, geos, proj, numpy,
+  gdal, pillow, matplotlib, pyepsg, pykdtree, scipy, owslib, fiona
+}:
+
+buildPythonPackage rec {
+  pname = "Cartopy";
```

```suggestion
  pname = "cartopy";
```

See https://nixos.org/nixpkgs/manual/#sec-package-naming